### PR TITLE
Ft pep8 warnings

### PR DESCRIPTION
**Pull request by dmigwi**

### CODE REVIEW

**Comment by dmigwi** [comment]
not important

File: extras/bench/test_Simple.py [hunk 1]
```diff
@@ -23,16 +22,19 @@ def test_simple(self):
         server_url = self.server_url
 
         # Exercises
-        self.get(server_url + '/en/exercise/overview/',
-                 description='Get exercise overview')
-        self.get(server_url + '/en/exercise/muscle/overview/',
-                 description='Get muscle overview')
+        self.get(
```
Comment (mfalade) [inline extras/bench/test_Simple.py:25]: 👍  on keeping a consistent pattern.

File: extras/dummy_generator/generator.py [hunk 2]
```diff
@@ -75,11 +73,11 @@
                          default='auto',
                          help='Gym to assign the users to. Allowed values:'
                          'auto, none, <gym_id>. Default: auto')
-help = 'What country the generated users should belong to. Default: Germany'
+help_to = 'What country the generated users should belong to. Default: Germany'
```
Comment (mfalade) [inline extras/dummy_generator/generator.py]: How about you name this something like `country_help`

File: extras/dummy_generator/generator.py [hunk 3]
```diff
@@ -108,10 +106,10 @@
 
 # Session options
 session_parser = subparsers.add_parser('sessions', help='Create sessions')
-help = 'Impression for the sessions, default: random'
+help_session = 'Impression for the sessions, default: random'
 session_parser.add_argument('impression_sessions',
                             action='store',
-                            help=help,
+                            help=help_session,
```
Comment (mfalade) [inline extras/dummy_generator/generator.py]:
``` python
impression_session_help
```

File: extras/dummy_generator/generator.py [hunk 4]
```diff
@@ -121,14 +119,14 @@
                            action='store',
                            help='Number of weight entries to create per user',
                            type=int)
-help = 'Add to the specified user-ID, not all existing users'
+help_user_id = 'Add to the specified user-ID, not all existing users'
```
Comment (mfalade) [inline extras/dummy_generator/generator.py:123]:
``` python
add_to_user_help
```

File: extras/dummy_generator/generator.py [hunk 5]
```diff
@@ -452,10 +448,10 @@
 
             creation_date = datetime.date.today() - datetime.timedelta(days=i)
             if creation_date not in existing_entries:
-                entry = WeightEntry(user=user,
-                                    weight=args.base_weight + 0.5 *
-                                    i + random.randint(1, 3),
-                                    date=creation_date)
+                entry = WeightEntry(
+                    user=user,
+                    weight=args.base_weight + 0.5 * i + random.randint(1, 3),
```
Comment (mfalade) [inline extras/dummy_generator/generator.py]: It's cleaner to have `weight` defined  outside like 

``` python
weight = args.base_weight + 0.5 * i + random.randint(1, 3)
```

Then have entry equal

``` python
entry = WeightEntry(
    user=user,
    weight=weight,
    date=creation_date
)
```

**Pull request closed without merging**